frontend/src/features/domain-pack-summary-read/ui/SummaryJsonCard.module.css

```
@@ -0,0 +1,95 @@
+}
+
+.key {
+  font-family: var(--font-mono);
+  font-size: var(--font-size-label);
+  color: var(--text-secondary);
+  white-space: nowrap;
+}
+
+.value {
+  color: var(--text-primary);
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)